styles.css

```diff
@@ -0,0 +1,90 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+
+}
+.app {
+    width: 100vw;
+    height: 100vh;
+    background: url(images/background.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.nav-bar-list {
+    display: flex;
+    gap: 32px;
+   align-items: center;
+}
+.nav-bar-list-right {
+    display: flex;
+    gap: 12px;
+   align-items: center;
+}
+.nav-bar {
+    max-width: 1440px;
+    height: 72px;
+    display: flex;
+    align-items: center;
+    padding: 0 20px;
+    justify-content: space-between;
```

Comment:
It would be better if you can use justify-content:space-around;

Reply:
the thing is, If space around is used, then imaginary margin within which the design is contained would be no more.
with space between the everything flushes to the left and right ends properly.